Exiled.Events/EventArgs/TriggeringTeslaEventArgs.cs

```diff
@@ -21,14 +21,12 @@ public class TriggeringTeslaEventArgs : EventArgs
         /// </summary>
         /// <param name="player"><inheritdoc cref="Player"/></param>
         /// <param name="teslaGate"><inheritdoc cref="Tesla"/></param>
-        /// <param name="isInHurtingRange"><inheritdoc cref="IsInHurtingRange"/></param>
-        /// <param name="isTriggerable"><inheritdoc cref="IsTriggerable"/></param>
-        public TriggeringTeslaEventArgs(Player player, TeslaGate teslaGate, bool isInHurtingRange, bool isTriggerable)
```

Comment:
Why would we remove these parameters, just to re-run the same methods for them? It's inefficienct, especially considering each of these methods preforms a Math.sqrt operation.

Reply:
it's not for re-run i have thinking of only run it in the event and don't run this method in the normal method

Reply:
Oh I misread the patch part, my bad

Reply:
yes it's the reason why i have made the prefix just for explain how it's could be ^^'